### PR TITLE
T31871 breadcrumb

### PR DESCRIPTION
**Pull request by manuq**
https://phabricator.endlessm.com/T31871

### CODE REVIEW

File: template-ui/src/components/Breadcrumb.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<template>
+  <div
+    v-if="!isBreadcrumbDisabled"
```
Comment (dylanmccall) [inline template-ui/src/components/Breadcrumb.vue:3]: One nitpick: "isXDisabled" sounds like we're checking some configuration to me, and double negative is :(

Perhaps…

```suggestion
    v-if="hasAncestors"
```

**Review by dylanmccall — CHANGES_REQUESTED** [review]
One small thing, but otherwise I like it. This is definitely way better :)

File: template-ui/src/components/Breadcrumb.vue (hunk 1 above)
Comment (manuq) [reply template-ui/src/components/Breadcrumb.vue:3]: Good point, I'm carrying the old logic here.

Comment (manuq) [reply template-ui/src/components/Breadcrumb.vue]: I realized that we can just check `v-if="node.ancestors"` directly, without introducing a new method.

**Review by dylanmccall — APPROVED** [review]

**Pull request merged**